openfecwebapp/templates/legal-advisory-opinion.html

```diff
@@ -33,7 +33,7 @@ <h4 class="sidebar__title">Cited in this opinion</h4>
     <section class="content__section">
       <h2 class="t-ruled--bold t-ruled--bottom">Documents</h2>
       <div class="content__section">
-        <a class="button button--cta button--document button--lg" href="{{ advisory_opinion.url }}">Final opinion</a>
+        <a class="button button--cta button--document button--lg" href="{{ advisory_opinion.url }}"> View {{ advisory_opinion.category }}</a>
```

Comment:
I also added "View" (to match the mockup).  Should I just send the category to all lower case? `View final opinion` or `View withdrawl of request`?